1080i/Includes_Search_Filter.xml

```diff
@@ -644,23 +644,23 @@
             </focusedlayout>
             <content>
                 <item>
-                    <label>All items</label>
+                    <label>$LOCALIZE[593]</label>
```

Comment:
593 is just "All" not "All items". It's the wrong label. 
It's used for setting buttons e.g. "Shuffle: All" "Lock settings: All" "In-progress: All"

There is an "All items" label in the skin 31382 for this purpose:
https://github.com/jurialmunkey/skin.arctic.fuse/blob/221d025260640c246633477111d85968cf831bed/language/resource.language.en_gb/strings.po#L1333-L1336

Reply:
Thanks, fixed.